.gitignore

```diff
@@ -22,3 +22,5 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+.env
```

Comment:
- [ ] Your frontend app depends on a `.env` file that is not being shared to the repo.  This means, your project needs special procedures to be run correctly:

![image](https://github.com/hienphan0111/Mentor-front-end-app/assets/57421823/4adc7cb3-b97f-4a62-9a62-38f82b7385cf)

![image](https://github.com/hienphan0111/Mentor-front-end-app/assets/57421823/d73b27fe-9e23-49e8-8162-3cb78e383691)

![image](https://github.com/hienphan0111/Mentor-front-end-app/assets/57421823/4d947602-4a80-46d0-8268-a78297545d7a)

Therefore, if I try to create a new user:

![image](https://github.com/hienphan0111/Mentor-front-end-app/assets/57421823/019418c6-8bac-406c-968c-0799fb96820b)

![image](https://github.com/hienphan0111/Mentor-front-end-app/assets/57421823/1c9ca8fc-1cc8-4576-99ec-6489c0d33907)

![image](https://github.com/hienphan0111/Mentor-front-end-app/assets/57421823/619be663-5036-405a-bc85-2e9ac3156ac9)

Kindly, either push the `.env` file, or include in your readme specific instructions about how to create one with the correct data to be able to run your project correctly.  Also, if there is additional special procedures to run your project, kindly include those instructions in your readme file.

That way, all your users will be able to run your project smoothly. 💪 

_note: This means I couldn't get access to your app functionality to review it.  This will be done in the next review._